man/rebase-helper.1

```diff
@@ -50,5 +50,8 @@ output is more verbose
 \fB\-w\fR, \fB\-\-keep\-workspace\fR
 select the choice if you want to keep workspace directory after finishing
 
+.TP
+\fB\-\-build\--tries\fR=\fIBUILD_RETRIES\fR
+specify count of builds in case of failure. Rebase-helper tries to solve some build issues automatically.
```

Comment:
I propose the following text or something similar: "specifies number of retries rebase-helper will do in case of failed build. If build fails, rebase-helper analyzes the build log, tries to fix the issue if possible and retries the build."